src/engraving/dom/scorefile.cpp

```diff
@@ -123,11 +124,13 @@ bool Score::loadStyle(muse::io::IODevice& dev, bool ign, bool overlap)
     MStyle st = style();
     if (st.read(&dev, ign)) {
         undo(new ChangeStyle(this, st, overlap));
-        return true;
+        success = true;
+    } else {
+        LOGE() << "The style data is not compatible with this version of MuseScore Studio.";
     }
 
-    LOGE() << "The style data is not compatible with this version of MuseScore Studio.";
-    return false;
+    dev.close();
```

Comment:
in such cases it is better to use
```
    if (!dev.open(IODevice::ReadOnly)) {
        LOGE() << "The style data is not available.";
        return false;
    }

    DEFER {
         dev.close();
    };
```
but it is ok

Reply:
I was thinking to create a wrapper, didn't know we have one, thanks for telling! Ok, I'll not redo it but will keep in mind for next time